integrations/github/jwt.go

```diff
@@ -100,21 +100,24 @@ func (i integration) getConnection(ctx context.Context) (map[string]string, erro
 func (i integration) NewClientWithAppJWTFromGitHubID(appID int64) (*github.Client, error) {
 	// Shared transport to reuse TCP connections.
 	tr := http.DefaultTransport
+	entURL, err := enterpriseURL()
```

Comment:
enterpriseURL? in many cases I do prefer short names, but in this case I would advice to use the full one

Reply:
done